### PR TITLE
Installation for myBinder

### PR DESCRIPTION
**Pull request by amnona**
added the mybinder folder with the conda and post-installation requirements.
Now can paste to [https://mybinder.org/](url) and work remotely there :)

### CODE REVIEW

File: calour/analysis.py [hunk 1]
```diff
@@ -128,10 +128,17 @@ def diff_abundance(exp: Experiment, field, val1, val2=None, method='meandiff', t
...
         the desired FDR control level
-    numperm : int
+    numperm : int (optional)
         number of permutations to perform
+    fdr_method : str (optional)
+        The method used to control the False Discovery Rate. options are:
+            'dsfdr' : the discrete FDR control method
+            'bhfdr' : Benjamini-Hochberg FDR method
+            'byfdr' : Benjamini-Yekutielli FDR method
+            'filterBH' : Benjamini-Hochberg FDR method with filtering
```
Comment (RNAer) [inline calour/analysis.py]: how is the filtering done?

File: binder/environment.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - scikit-learn
+  - scikit-bio>=0.5.1
+  - biom-format
+  - statsmodels
+  - ipywidgets
+  - numpydoc
+  - sphinx
+  - sphinx_bootstrap_theme
+  - biom-format
+  - scikit-bio
```
Comment (RNAer) [inline binder/environment.yml]: how does the binder dir work? just curious, could you point me to a tutorial/description?

Comment (amnona) [reply binder/environment.yml]: it creates a docker file for the repo installation and runs a new instance for each user
to specify the installation requirements, you supply conda/pip/batch files (like for travis)
see [here](https://mybinder.readthedocs.io/en/latest/using.html#environment-yml)

File: calour/analysis.py (hunk 1 above)
Comment (amnona) [reply calour/analysis.py]: ? don't understand the question ?
is it related to this PR or general?

**Comment by amnona** [comment]
adressed 1st comment
didn't understand the second :)

File: calour/analysis.py (hunk 1 above)
Comment (RNAer) [reply calour/analysis.py]: i just don't understand what does 'filterBH' do based on the doc.

**Comment by amnona** [comment]
ok added small explanation for filterBH in docstring
will provide better explanation in a new API documentation PR (which will handle documenting parameters passed to other functions - so will use better dsFDR documentation and pass-through to calour function docs)

**Pull request merged**